includes/people-functions.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+/**
+ * Updates the people_group custom taxonomies labels
+ **/
+function people_group_labels( $labels ) {
```

Comment:
Functions within plugins should be wrapped in a `function_exists` check or else be within a uniquely named class (that is itself wrapped in a `class_exists` check) to avoid naming conflicts.